### PR TITLE
added single products page

### PR DESCRIPTION
**Pull request by prathmeshjagtap**
Added Single products Page
user can add to cart and add to wishlist from single products page

### CODE REVIEW

**Review by KanekarSnehal — APPROVED** [review]
Code looks well structured
Like the use of useLocation for navigating

File: src/frontend/pages/singleProduct/SingleProduct.jsx [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+							<div className="current__price">
+								<i className="fas fa-rupee-sign fa-1x"></i>
+								<span>{singleProduct?.price}</span>
+							</div>
+							<div className="previous__value">
+								<i className="fas fa-rupee-sign fa-1x"></i>
+								<span>{singleProduct?.previousPrice}</span>
+							</div>
+						</div>
+
```
Comment (tanishk15sharma) [inline src/frontend/pages/singleProduct/SingleProduct.jsx]: Use prettier , dont leave black space .

**Review by tanishk15sharma — APPROVED** [review]
Functionality is working fine .

File: src/frontend/pages/singleProduct/SingleProduct.jsx (hunk 1 above)
Comment (prathmeshjagtap) [reply src/frontend/pages/singleProduct/SingleProduct.jsx]: yes fixed this

**Pull request merged**